체육복.js

```diff
@@ -0,0 +1,23 @@
+function solution(n, lost, reserve) {
+    let arr1
+    arr1 = lost.filter(x => reserve.includes(x));
```

Comment:
```suggestion
    let arr1 = lost.filter(x => reserve.includes(x));
```
선언 후 초기화된 값이 없다면, 하나로 사용하는게 좋을 것 같습니다.